client/admin/pages/api/form.ts

```diff
@@ -0,0 +1,42 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import axios from "axios";
+
+const handelSubmit = async ({
+  form,
+}: {
+  form: { email: string; password: string };
+}) => {
+  try {
+    const { data } = await axios.post(
+      `${process.env.NEXT_PUBLIC_API_URL}/login`,
```

Comment:
This runs on the server, `NEXT_PUBLIC_API_URL` would be `undefined`. Was this tested?